gravitee-apim-rest-api/gravitee-apim-rest-api-service/src/test/java/io/gravitee/apim/core/integration/use_case/CreateIntegrationUseCaseTest.java

```diff
@@ -120,15 +176,49 @@ void should_create_new_integration() {
             );
     }
 
+    @Test
+    void should_create_new_primary_owner_membership() {
+        //Given
+        givenExistingUsers(
+            List.of(BaseUserEntity.builder().id(USER_ID).firstname("Jane").lastname("Doe").email("jane.doe@gravitee.io").build())
+        );
+        roleQueryServiceInMemory.resetSystemRoles(ORGANIZATION_ID);
+        var input = new Input(IntegrationFixture.anIntegration(), AUDIT_INFO);
+
+        //When
+        usecase.execute(input);
+
+        //Then
+        assertThat(membershipCrudServiceInMemory.storage())
+            .hasSize(1)
+            .containsExactly(
```

Comment:
The `containsExactly()` will check the size and will give more usefull error message in cas of issue, I think we can remove the `hasSize(1)`